container.go

```diff
@@ -101,28 +102,29 @@ type ContainerRequest struct {
 	Cmd             []string
 	Labels          map[string]string
 	Mounts          ContainerMounts
-	Tmpfs           map[string]string
+	Tmpfs           map[string]string // Deprecated: Use PreCreationHook instead
 	RegistryCred    string
 	WaitingFor      wait.Strategy
 	Name            string // for specifying container name
 	Hostname        string
-	ExtraHosts      []string
-	Privileged      bool                // for starting privileged container
-	Networks        []string            // for specifying network names
-	NetworkAliases  map[string][]string // for specifying network aliases
-	NetworkMode     container.NetworkMode
-	Resources       container.Resources
-	Files           []ContainerFile // files which will be copied when container starts
-	User            string          // for specifying uid:gid
-	SkipReaper      bool            // indicates whether we skip setting up a reaper for this
-	ReaperImage     string          // alternative reaper image
-	AutoRemove      bool            // if set to true, the container will be removed from the host when stopped
-	AlwaysPullImage bool            // Always pull image
-	ImagePlatform   string          // ImagePlatform describes the platform which the image runs on.
-	Binds           []string
-	ShmSize         int64    // Amount of memory shared with the host (in bytes)
-	CapAdd          []string // Add Linux capabilities
-	CapDrop         []string // Drop Linux capabilities
+	ExtraHosts      []string                                                          // Deprecated: Use PreCreationHook instead
```

Comment:
1st level in Java. But we don't see any modules using it, not sure how much used by our users and in which use cases.